src/instrumentsscene/view/staffcontroltreeitem.cpp

```diff
@@ -51,4 +51,7 @@ void StaffControlTreeItem::appendNewItem()
     staff->setDefaultClefType(part->instrument()->clefType(lastStaffIndex));
 
     masterNotation()->parts()->appendStaff(staff, m_partId);
+    if (this->parentItem()->isSelected()) {
```

Comment:
For better readability, we prefer not unnecessarily using `this->`:
```suggestion
    if (parentItem()->isSelected()) {
```